### DISCUSSION TITLE
Forgotten gmail password

### DISCUSSION THREAD

**Discussion by Nill62065** [community/community#38701] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

[removed by moderator] that gmail account forgotten password please help me

**Comment by ossanna16** [comment]
Hi @Nill62065,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Gmail. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
